orchestrator_service/src/main/java/com/override/orchestrator_service/service/KeywordService.java

```diff
@@ -44,4 +42,8 @@ public void removeCategoryFromKeywordByTransactionId(UUID transactionId) {
                 transaction.getAccount().getId());
         keywordRepository.removeCategoryId(keywordId);
     }
+
+    public List<Keyword> findAllByKeywordId_AccountId(OverMoneyAccount overMoneyAccount){
```

Comment:
название не отражает сути: поиск не по полю составного ключа. сервисный метод ищет по аккаунту, судя по сигнатуре.
```suggestion
    public List<Keyword> findAllByOverMoneyAccount(OverMoneyAccount overMoneyAccount){
```